tests/contracts/filtered-call/lib.rs

```diff
@@ -0,0 +1,116 @@
+#![cfg_attr(not(feature = "std"), no_std, no_main)]
+
+/// This is an example contract used for unit testing purposes.
+/// It is intended to ensure that the call filter for the Pop API chain
+/// extension works as expected.
+/// An Ok(()) returned from `get_filtered` means that the filter successfully
+/// BLOCKED the call. An Err() means that the filter did not block the call,
+/// which is incorrect behavior.
+/// The pop-api crate is not used because it does not support invalid calls ;)
```

Comment:
I don't get this line. It does not support *filtered calls?

True, but is that not what this test is for? Devs using pop api and not being able to use this call in their contracts?

Reply:
The call defined in the contract is filtered (blocked) by the call filter. We do not want anything in the pop-api that is not a valid call (i.e. filtered)